test/inductor/test_pattern_matcher.py

```diff
@@ -24,17 +24,19 @@
     stable_topological_sort,
 )
 from torch._inductor.test_case import run_tests, TestCase
-from torch._inductor.utils import get_gpu_shared_memory, run_and_get_code
+from torch._inductor.utils import run_and_get_code
 from torch._inductor.virtualized import V
 from torch.testing import FileCheck
 from torch.testing._internal.common_cuda import SM80OrLater
 from torch.testing._internal.common_utils import IS_LINUX, LazyVal, skipIfRocm
 from torch.testing._internal.inductor_utils import HAS_CUDA
 from torch.utils import _pytree as pytree
 
-
-is_A100 = LazyVal(
-    lambda: torch.cuda.is_available() and get_gpu_shared_memory() == 166912
+# NVIDIA A100-SXM4-40GB
+is_a100 = LazyVal(
```

Comment:
What do you think about renaming `is_a100` to `is_a100_linux` or something else taht more accurately reflects the constraints of the test?

Reply:
Sure, that seems like an easy tweak